.github/workflows/modmesh_viewer.yml

```diff
@@ -15,7 +15,7 @@ jobs:
 
     strategy:
         matrix:
-          os: [ubuntu-22.04]
+          os: [ubuntu-22.04, macos-latest]
```

Comment:
You have changed it to `macos-12` in #116 but here it goes back to `macos-latest`.  Perhaps an overlook?

Reply:
Yes, it is an overlook and I have already fixed it.

Reply:
LGTM.  I will merge once the CI passes.